### PR TITLE
OC-9354: SdvDTO created for the new API

### PR DESCRIPTION
**Pull request by suthirr-openclinica**

### CODE REVIEW

File: src/main/java/org/akaza/openclinica/controller/dto/SdvDTO.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+public class SdvDTO {
+    private int participantId;
+    private String siteName;
+    private String eventName;
+    private Date eventStartDate;
+    private String sdvRequirement;
+    private String sdvStatus;
+    private String formName;
+    private String formStatus;
+    private Date itemsLastVerifiedDate;
```
Comment (svadla-oc) [inline src/main/java/org/akaza/openclinica/controller/dto/SdvDTO.java]: rename this to "lastVerifiedDate"

File: src/main/java/org/akaza/openclinica/controller/dto/SdvDTO.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
+package org.akaza.openclinica.controller.dto;
+
+import java.util.Date;
+import java.util.List;
+
+public class SdvDTO {
+    private int participantId;
```
Comment (svadla-oc) [inline src/main/java/org/akaza/openclinica/controller/dto/SdvDTO.java]: this should be a string

File: src/main/java/org/akaza/openclinica/controller/dto/SdvItemDTO.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
+package org.akaza.openclinica.controller.dto;
+
+import java.util.Date;
+
+public class SdvItemDTO {
+    private String name;
+    private String briefDesc;
```
Comment (svadla-oc) [inline src/main/java/org/akaza/openclinica/controller/dto/SdvItemDTO.java]: it's always better to have full name. rename this to briefDescription

Comment (suthirr-openclinica) [reply src/main/java/org/akaza/openclinica/controller/dto/SdvItemDTO.java]: Updated as per the Comments

File: src/main/java/org/akaza/openclinica/controller/dto/SdvDTO.java (hunk 2 above)
Comment (suthirr-openclinica) [reply src/main/java/org/akaza/openclinica/controller/dto/SdvDTO.java]: Updated as per the Comments

File: src/main/java/org/akaza/openclinica/controller/dto/SdvDTO.java (hunk 1 above)
Comment (suthirr-openclinica) [reply src/main/java/org/akaza/openclinica/controller/dto/SdvDTO.java]: Updated as per the Comments

**Comment by svadla-oc** [comment]
👍

**Pull request merged**